src/sugar3/graphics/style.py

```diff
@@ -156,52 +156,78 @@ def zoom(units):
     return int(ZOOM_FACTOR * units)
 
 
-ZOOM_FACTOR = _compute_zoom_factor()
+ZOOM_FACTOR = _compute_zoom_factor()  #: scale factor, as float (eg. 0.72, 1.0)
 
-DEFAULT_SPACING = zoom(15)
-DEFAULT_PADDING = zoom(6)
-GRID_CELL_SIZE = zoom(75)
-LINE_WIDTH = zoom(2)
+DEFAULT_SPACING = zoom(15)  #: Spacing is placed in-between components
+DEFAULT_PADDING = zoom(6)  #: Padding is placed around a ui component
+GRID_CELL_SIZE = zoom(75)  #: Grid cells are the fundamental measure for sugar
+LINE_WIDTH = zoom(2)  #: Thickness of a separator line
 
+#: icon that fits within a grid cell
 STANDARD_ICON_SIZE = zoom(55)
+#: small icon, used in palette menu items
 SMALL_ICON_SIZE = zoom(33)
+#: larger than standard
 MEDIUM_ICON_SIZE = zoom(55 * 1.5)
+#: larger than medium, used in journal empty view
 LARGE_ICON_SIZE = zoom(55 * 2.0)
+#: larger than large, used in activity pulsing launcher icon
 XLARGE_ICON_SIZE = zoom(55 * 2.75)
 
 if 'org.sugarlabs.font' in Gio.Settings.list_schemas():
     settings = Gio.Settings('org.sugarlabs.font')
+    #: User's preferred font size
     FONT_SIZE = settings.get_double('default-size')
+    #: User's preferred font face
     FONT_FACE = settings.get_string('default-face')
 else:
+    #: User's preferred font size
     FONT_SIZE = 10
-    FONT_FACE = 'Sans Serif' 
+    #: User's preferred font face
+    FONT_FACE = 'Sans Serif'
 
+#: Normal font
 FONT_NORMAL = Font('%s %f' % (FONT_FACE, FONT_SIZE))
+#: Bold font
 FONT_BOLD = Font('%s bold %f' % (FONT_FACE, FONT_SIZE))
+#: Height in pixels of normal font
 FONT_NORMAL_H = zoom(24)
+#: Height in pixels of bold font
 FONT_BOLD_H = zoom(24)
 
+# These constants are unused, as they are for the old style toolbox design
```

Comment:
> are unused

They are used by src/sugar3/graphics/toolbarbox.py and src/sugar3/graphics/toolbox.py.

> old style toolbox design

`from sugar3.graphics.toolbarbox' occurs 65 times in the activities being shipped by OLPC, so it doesn't seem very old.  The constants themselves haven't been used by the activities as far as I can see.  Are you not documenting them here because of some API deprecation?